dev/MRTCore/TestApps/README.md

```diff
@@ -0,0 +1,41 @@
+Instructions for building the sample apps in this folder
+
+1. Temporary step:
+Remove the script code below from the build\Microsoft.WinUI.AppX.targets file in the WinUI NuGet package**.
+
+  <!-- In the case of a WinUI Desktop project, we want to include the AppxPackage targets file to get a PRI file generated,
+        but since we aren't actually packaging ourselves in an AppX file, we'll empty out our AppX recipe file properties. -->
+  <Import Project="$(MsAppxPackageTargets)" Condition="'$(WindowsAppContainer)' != 'true' and '$(MSBuildProjectExtension)'!='.wapproj'" />
+  <PropertyGroup Condition="'$(WindowsAppContainer)' != 'true'">
+    <AppxUploadPackageRecipe />
+    <AppxPackageRecipe />
+  </PropertyGroup>
+
+** Depending on whether the app is a C# or C++ one, the package will be in different locations.
+- In the case of C#, the app will use the package installed in the central location (e.g., C:\Users\rohanp.REDMOND\.nuget\packages).
+- In the case of C++, the package will be installed in the solution folder.
+
+2. Install the Microsoft.ApplicationModel.Resources NuGet package.
+    a. Connect to the following feed: https://mscodehub.visualstudio.com/mrtcore/_packaging?_a=feed&feed=MRTCore.
```

Comment:
Adding a note that this is a temporary location and will soon move is probably helpful.